### ISSUE TITLE
set purge_configs to true

### ISSUE THREAD

**Issue by smetj** [marcomusso/puppet-cachet#5]
https://github.com/marcomusso/puppet-cachet/blob/master/manifests/apache.pp#L12

Set default value to true
I'd expect that applying the module gives a working apache.
If you don't set this to true you end up having "port already in use" and apache not starting

**Comment by marcomusso** [comment]
Yep, it was a conservative choice that can be overridden in hiera. But I can see why you want to change the default...

**Comment by marcomusso** [comment]
The reason for not purging configs is that you'll remove the php.conf installed by the php rpm.
